torch/csrc/jit/script/sugared_value.h

```diff
@@ -268,7 +269,9 @@ struct FunctionValue : public SugaredValue {
   }
 
  private:
-  std::shared_ptr<Function> callee_;
+  Function* callee_;
```

Comment:
Can't these two things just be a StrongFunctionPtr?

Reply:
We don't always need to construct an owning FunctionValue, and soon there will be owning FunctionValues. So I left them decomposed for now.